source/docs/v3/tenant-aware-commands.blade.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Tenant-aware commands
+description: Tenant-aware commands
+extends: _layouts.documentation
+section: content
+---
+
+# Tenant-aware commands {#tenant-aware-commands}
+
+Even though [`tenants:run`]({{ $page->link('console-commands#run') }}) lets you run arbitrary artisan commands for tenants, you may want to have strictly tenant commands.
+
+To make a command tenant-aware, utilize the `TenantAwareCommand` trait:
+```php
+class MyCommand extends Command
+{
+    use TenantAwareCommand;
+}
+```
+
+However, this trait requires you to implement a `getTenants()` method that returns an array of `Tenant` instances.
+
+If you don't want to implement the options/arguments yourself, you may use one of these two traits:
+- `HasATenantsOption` - accepts multiple tenant IDs, optional -- by default the command is executed for all tenants
```

Comment:
Is this the name of the trait in v3?

Reply:
Just double-checked, and yeah, it is

<img width="775" alt="image" src="https://user-images.githubusercontent.com/34375937/189277011-52592b6d-9d44-407b-a0dc-f80de35511e6.png">